packages/azure-kusto-data/test/cloudSettingsTest.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import assert from "assert";
+
+import { CloudSettings } from "../src/cloudSettings.js";
+
+describe("CloudSettings.getAuthMetadataEndpointFromClusterUri", () => {
+    it("valid input", () => {
+        assert.strictEqual(
+            CloudSettings.getAuthMetadataEndpointFromClusterUri("https://statusreturner.azurewebsites.net/"),

```

Comment:
Weird that this works - since the method isn't static and uses a member. 
I think make the method and METADATA_ENDPOINT static

Reply:
Resolved